loris/terraform/loris.tf

```diff
@@ -0,0 +1,43 @@
+module "loris_service" {
+  source = "service"
+
+  namespace = "loris-delta"
+
+  certificate_domain = "api.wellcomecollection.org"
```

Comment:
???

Reply:
This is a multi domain certificate (also covers iiif.wc & iiif-origin.wc) not actually a change from existing tf.